bower.json

```diff
@@ -31,9 +31,10 @@
     "ckeditor": "~4.5.4",
     "angular-ckeditor": "~1.0.0",
     "roboto-condensed": "~0.3.0",
-    "open-sans-fontface": "https://github.com/OpenSlides/open-sans.git#1.4.2.post1"
+    "open-sans-fontface": "https://github.com/OpenSlides/open-sans.git#1.4.2.post1",
+    "angular-pdf": "~1.3.0"
   },
   "resolutions": {
-    "angular": "^1.2.x"
+    "angular": ">= 1.4.9"
```

Comment:
~1.4.9 will match all 1.4.x versions but will miss 1.5.0.
^1.4.9 will match any 1.x.x release including 1.5.0, but will hold off on 2.0.0.
What we need is 1.4.9+ (>=1.4.9).

And yes. We should do `>=1.4.9 <1.5`